dbcrossbar/src/clouds/gcloud/client.rs

```diff
@@ -222,6 +222,68 @@ impl Client {
         }
     }
 
+    /// Make an HTTP POST request and return the response.
+    async fn post_helper<Body>(
+        &self,
+        url: &Url,
+        body: &Body,
+    ) -> Result<reqwest::Response, ClientError>
+    where
+        Body: fmt::Debug + Serialize + std::marker::Sync,
```

Comment:
If you require `Sync`, you might as well require `Send`. Also, these are in scope by default.

```suggestion
        Body: fmt::Debug + Serialize + Sync + Send,
```

Reply:
updated